schemas/generate_json_schema.py

```diff
@@ -0,0 +1,134 @@
+"""
```

Comment:
Eeeeehhhhhh I wish we didn't have to do this.  There's not something else that works off the shelf?

Reply:
So yea, I agree. I think there may be some options, but nothing that jumped out at me when doing some initial searching. I also know Beth was interested in pursuing something like Zod and I don't really have the time to evaluate things like that at the moment. This is becoming more of an issue for data so I saw this as a bit of a stopgap, but taking inspiration from tooling we already use.

Reply:
Okey doke fair enough.  Let's do it for now to unblock things and maybe we can find a better solution in the future.  Thanks for taking this on @mikewilli 🙏